lib/validate/latest.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/**
+/*
```

Comment:
not the best way to hide documentation but I don't really feel like wrangling dox today